docs/book/src/topics/external-resource-gc.md

```diff
@@ -1,6 +1,6 @@
 # External Resource Garbage Collection
 
-- **Feature status:** Experimental
+- **Feature status:** Stable
```

Comment:
We only have two statuses `Stable` and `Experimental`, correct? I'm checking, just in case this should be `Beta` instead.

Reply:
Yes thats right, just 2 statuses from the docs at present.